src/lang/lang-middleware.ts

```diff
@@ -0,0 +1,54 @@
+import {AppContext} from '@gravity-ui/nodekit';
+import acceptLanguage from 'accept-language-parser';
+import type {Express} from 'express';
+import {setLang} from './set-lang';
+
+export function setupLangMiddleware(appCtx: AppContext, expressApp: Express) {
+    const config = appCtx.config;
+    const languageConfig = config.language;
+    if (!languageConfig) {
+        return;
+    }
+
+    const {defaultLang, allowLanguages, langQueryParamName} = languageConfig;
+    if (allowLanguages && allowLanguages.length > 0 && defaultLang) {
```

Comment:
Maybe we can use return if one of this is null

Reply:
yep, fixed it